solutions/cses-1158.mdx

```diff
@@ -35,7 +35,7 @@ int main() {
 		cin >> pages[i];
 	}
 
-	vector<vector<int>> dp(n + 1, vector<int>(x + 1));
+	vector<vector<int>> dp(n+1, vector<int>(x+1));
```

Comment:
wha why would you _remove_ spaces?

Reply:
you said remove spaces, so i did that in places that were like that. anyways, i added them back